src/sonic-yang-models/yang-models/sonic-bgp-peerrange.yang

```diff
@@ -0,0 +1,64 @@
+module sonic-bgp-peerrange {
+    namespace "http://github.com/Azure/sonic-bgp-peerrange";
+    prefix pr;
+    yang-version 1.1;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC BGP Peer Range YANG";
+
+    revision 2022-02-24 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-bgp-peerrange {
+        container BGP_PEER_RANGE {
+            list BGP_PEER_RANGE_LIST {
+                key "peer_range_name";
+
+                leaf peer_range_name {
+                    type string;
+                    description "Peer range name";
+                }
+
+                leaf name {
+                    type string;
+                    must "(current() = current()/../peer_range_name)" {
+                        error-message "Invalid name";
+                    }
+                }
+
+                leaf src_address {
```

Comment:
>src_address [](http://example.com/codeflow?start=21&length=11)

Is it true that src_address must be one of the loopback IP? If yes, let's enforce it.